src/js/vre/chapter31/config/form.js

```diff
@@ -176,23 +176,13 @@ const formConfig = {
               'ui:title': 'Are you working?',
               'ui:widget': 'yesNo'
             },
-            employer: {
+            employer: _.merge({
               'ui:title': 'Employer name',
               'ui:required': (formData) => formData['view:isWorking'],
-              ...expandIfWorking
-            },
-            jobDuties: {
-              'ui:title': 'Job duties',
-              ...expandIfWorking
-            },
-            monthlyIncome: {
```

Comment:
This doesn't assign the correct class to the element. Just refactored everything to use `_.merge`